Tests/Indicators/SumTests.cs

```diff
@@ -52,5 +54,19 @@ public class SumTests {
             sum.Update(time.AddDays(1), 1);
             Assert.AreEqual(sum.Current.Value, 1m);
         }
+
+        [Test]
+        public void WarmsUpProperly()
+        {
+            var indicator = new Sum(2);
```

Comment:
no need to change -- this guy should likely be updated to follow the `CommonIndicatorTests` pattern and _if not_ maybe just a comment describing why it doesn't fit that pattern.